### PR TITLE
More styling

### PR DESCRIPTION
**Pull request by kel29**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -65,22 +83,41 @@ class App extends Component {
...
                 {...props}
-                handleLogIn={this.logIn}
+                handleLogin={this.login}
                 users={this.state.users}
                 addNewUser={this.addNewUser}
                 loggedIn={this.state.loggedIn}
               />}
             }/>
-            <div className="card shadow mt-3 p-3">
+            <div className="">
```
Comment (renburnett) [inline src/App.js]: do we need this div?

File: src/components/Message.js [hunk 2]
```diff
@@ -1,13 +1,29 @@
 import React from 'react'
 
 const Message = (props) => {
-  const findUser = () => {
+  const findMessageAuthor = () => {
```
Comment (renburnett) [inline src/components/Message.js:4]: love this rename. makes much more sense

File: src/App.js (hunk 1 above)
Comment (kel29) [reply src/App.js]: not at all. I'll remove it

**Review by ianhunterharold — APPROVED** [review]

File: src/containers/Sidebar.js [hunk 3]
```diff
@@ -8,27 +8,28 @@ class Sidebar extends PureComponent {
...
 
     const config = {
-      method: 'POST',      
+      method: 'POST',
       headers: {
         'Content-Type': 'application/json',
         'accept': 'application/json'
       },
       body: JSON.stringify({
-        topic: e.target.children[0].value
```
Comment (renburnett) [inline src/containers/Sidebar.js:28]: great refactor

**Review by renburnett — APPROVED** [review]

**Review by renburnett — APPROVED** [review]

**Pull request merged**